yt_dlp/extractor/ant1newsgr.py

```diff
@@ -0,0 +1,164 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+import urllib.parse
+
+from .common import InfoExtractor
+from ..utils import (
+    HEADRequest,
+    ExtractorError,
+    determine_ext,
+    merge_dicts,
+    unescapeHTML,
+)
+
+
+class Ant1NewsGrBaseIE(InfoExtractor):
+    @staticmethod
+    def _scale_thumbnails_to_max_width(formats, thumbnails, url_width_re):
+        _keys = ('width', 'height')
+        max_dimensions = max(
+            [tuple(format.get(k) or 0 for k in _keys) for format in formats],
+            default=(0, 0))
+        if not max_dimensions[0]:
+            return thumbnails
+        return [
+            merge_dicts(
+                {'url': re.sub(url_width_re, str(max_dimensions[0]), thumbnail['url'])},
+                dict(zip(_keys, max_dimensions)), thumbnail)
+            for thumbnail in thumbnails
+        ]
+
+    def _download_and_extract_api_data(self, video_id, netloc, cid=None):
+        url = f'{self.http_scheme()}//{netloc}{self._API_PATH}'
+        info = self._download_json(url, video_id, query={'cid': cid or video_id})
+        try:
+            source = info['url']
+        except KeyError:
+            raise ExtractorError('no source found for %s' % video_id)
+        formats, subs = self._extract_m3u8_formats_and_subtitles(
+            source, video_id, 'mp4') \
+            if determine_ext(source) == 'm3u8' else ([{'url': source}], {})
+        self._sort_formats(formats)
+        thumbnails = self._scale_thumbnails_to_max_width(
+            formats, [{'url': info['thumb']}], r'(?<=/imgHandler/)\d+')
+        return {
+            'id': video_id,
+            'title': info['title'],
```

Comment:
```suggestion
            'title': info.get('title'),
```
due to some recent changes, this is now more appropriate